.github/workflows/default.yml

```diff
@@ -45,6 +45,22 @@ jobs:
           command: fmt
           args: --manifest-path avail-subxt/Cargo.toml --check
 
+      - name: Install zepter
+        continue-on-error: true
+        run: |
+          cargo install zepter --locked --version 0.11.0 -q -f --no-default-features && zepter --version
+
+      - name: Check Avail toml formatting
+        # Not enforcing the toml formatting for now
+        continue-on-error: true
```

Comment:
Not enforcing?

Reply:
Yeah, the check is just for awareness purposes now, later maybe we can enforce it, wdyt? just to not annoy the devs on toml formatting

Reply:
IMHO we should enforce it. Otherwise we will need a periodically task to apply formatting.